### PR TITLE
Add the search() method

### PR DESCRIPTION
**Pull request by thekad**
Had to add the ability for the _call() method to receive a dict of params, the way the body is sent already makes it so we cannot send POST form fields but GET fields should be fine.

### CODE REVIEW

File: chronos/__init__.py [hunk 1]
```diff
@@ -146,7 +163,15 @@ def _call(self, url, method="GET", body=None, headers={}):
         while servers:
             server = servers.pop(0)
             endpoint = "%s%s" % (server, quote(url))
+            if params:
+                if method == 'GET':
+                    endpoint += '?%s' % (urlencode(params))
+                else:
```
Comment (solarkennedy) [inline chronos/__init__.py]: Why bother with the else that does nothing?

File: chronos/__init__.py [hunk 2]
```diff
@@ -146,7 +163,15 @@ def _call(self, url, method="GET", body=None, headers={}):
         while servers:
             server = servers.pop(0)
             endpoint = "%s%s" % (server, quote(url))
+            if params:
```
Comment (solarkennedy) [inline chronos/__init__.py]: How about `if params and method==get`?

**Comment by thekad** [comment]
@solarkennedy I made the changes you suggested and also added tests for the new feature. Let me know what you think.

**Comment by solarkennedy** [comment]
Looks good! Great job.

**Pull request merged**